docs/ref/forms/fields.txt

```diff
@@ -504,22 +504,32 @@ For each field, we describe the default widget used if you don't specify
     .. attribute:: input_formats
 
         A list of formats used to attempt to convert a string to a valid
-        ``datetime.datetime`` object.
+        ``datetime.datetime`` object, in addition to ISO 8601 formats.
 
-    If no ``input_formats`` argument is provided, the default input formats are::
+    The field always accepts strings in ISO 8601 formatted dates or similar
+    recognized by :func:`django.utils.dateparse.parse_datetime`. Some examples
+    are::
+
+        * '2006-10-25 14:30:59'
+        * '2006-10-25T14:30:59'
+        * '2006-10-25 14:30'
+        * '2006-10-25T14:30'
+        * '2006-10-25T14:30Z'
+        * '2006-10-25T14:30+02:00'
+        * '2006-10-25'
 
-        ['%Y-%m-%d %H:%M:%S',    # '2006-10-25 14:30:59'
-         '%Y-%m-%d %H:%M',       # '2006-10-25 14:30'
-         '%Y-%m-%d',             # '2006-10-25'
-         '%m/%d/%Y %H:%M:%S',    # '10/25/2006 14:30:59'
-         '%m/%d/%Y %H:%M',       # '10/25/2006 14:30'
-         '%m/%d/%Y',             # '10/25/2006'
-         '%m/%d/%y %H:%M:%S',    # '10/25/06 14:30:59'
-         '%m/%d/%y %H:%M',       # '10/25/06 14:30'
-         '%m/%d/%y']             # '10/25/06'
+    If no ``input_formats`` argument is provided, the default input formats are
+    taken from :setting:`DATETIME_INPUT_FORMATS` if :setting:`USE_L10N` is
+    ``False``, or from the active locale format ``DATETIME_INPUT_FORMATS`` key
+    if localization is enabled.
```

Comment:
This small clarification is valid also without this change so we can move it to a separate commit.